DEVELOPERS.md

```diff
@@ -536,31 +539,31 @@ job-server. Refer to the documentation of
 
 This interacts with the [JobRequestAPIList](jobserver/api/jobs.py#L239) (GET /job-requests/) view for reading JobRequests, and the [JobAPIUpdate](jobserver/api/jobs.py#L54) (POST /jobs/) view for updating the Jobs table. (Current as of 2024-09.)
 
-### [Airlock](https://github.com/opensafely-core/airlock)
+### [airlock](https://github.com/opensafely-core/airlock)
```

Comment:
- de-capitalizing "airlock" as that's the repo name and for consistency with how I've referred job-runner and job-server